tests/writer_util.py

```diff
@@ -14,60 +14,49 @@
 from __future__ import division
 from __future__ import print_function
 
-from datetime import datetime
-from decimal import Decimal
 import six
 from pytest import raises
 
-from amazon.ion.core import IonEventType, IonEvent
-from amazon.ion.core import IonType
+from amazon.ion.core import ION_STREAM_END_EVENT, IonEventType, IonEvent
 from amazon.ion.util import record
 from amazon.ion.writer import WriteEventType
 from tests import is_exception
 from tests import noop_manager
 
+_STREAM_END_EVENT = (ION_STREAM_END_EVENT,)
 
-_D = Decimal
-_DT = datetime
 
-_E = IonEvent
-_IT = IonType
-_ET = IonEventType
-
-_STREAM_END_EVENT = (_E(_ET.STREAM_END),)
-
-
-class _P(record('desc', 'events', 'expected')):
+class P(record('desc', 'events', 'expected')):
```

Comment:
I'm probably not being clear, but a name that is more than `P` is a good idea for test code that is to be *shared*.  For example, `WriterParameters`.  This can be idiomatically bound to `_P` or something in the individual test modules.